libs/components/forms/src/lib/modules/checkbox/checkbox.component.scss

```diff
@@ -5,6 +5,14 @@
   font-size: 14px;
 }
 
+.sky-checkbox-hint-text {
+  margin-top: var(--sky-margin-stacked-xs);
+}
+
+.sky-checkbox-form-extensions {
+  margin-left: calc(var(--sky-switch-size) + var(--sky-switch-margin));
+}
```

Comment:
extensions doesn't sound right in this context, maybe `sky-checkbox-form-margin`?

Reply:
updated